### DISCUSSION TITLE
Is seeding supposed to work with NordVPN?

### DISCUSSION THREAD

**Discussion by arcady-genkin** [haugene/docker-transmission-openvpn#2307] [Q&A]
Downloads work fine, but Transmission reports that the incoming port is closed using NordVPN.

Is this working properly for anyone using NordVPN?

Many thanks!

**Comment by edgd1er** [comment]
https://support.nordvpn.com/FAQ/1047408432/Do-you-offer-port-forwarding.htm#:~:text=Unfortunately%2C%20at%20the%20current%20moment,are%20using%20the%20same%20servers.
